app/assets/stylesheets/app/tags-show.scss

```diff
@@ -0,0 +1,57 @@
+$back-ground-tag-color: #f7f5fa;
+
+.tag-list {
+  margin: 5px;
+  text-align: center;
+  text-decoration: none;
+  min-width: 100px;
+  font-size: 14px;
+  border-radius: 15px;
+  padding: 2px 6px 2px 6px;
```

Comment:
**[stylelint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected longhand value '2px 6px 2px 6px' instead of '2px 6px'   shorthand-property-no-redundant-values